packages/rabbitmq/test/rabbitmqTxSubmitProvider.test.ts

```diff
@@ -0,0 +1,54 @@
+import { ProviderError, TxSubmitProvider } from '@cardano-sdk/core';
+import { RabbitMqTxSubmitProvider } from '../src';
+
+const BAD_CONNECTION_URL = new URL('amqp://localhost:1234');
+const GOOD_CONNECTION_URL = new URL('amqp://localhost');
+
+describe('RabbitMqTxSubmitProvider', () => {
+  let provider: TxSubmitProvider;
+
+  afterEach(() => provider?.close!());
+
+  describe('healthCheck', () => {
+    it('is not ok if cannot connect', async () => {
+      provider = new RabbitMqTxSubmitProvider(BAD_CONNECTION_URL);
+      const res = await provider.healthCheck();
+      expect(res).toEqual({ ok: false });
+    });
+
+    it('is ok if cannot connect', async () => {
+      provider = new RabbitMqTxSubmitProvider(GOOD_CONNECTION_URL);
+      // Called twice to cover AMQPWrapper.#connect() first line as well
```

Comment:
What happens in the first line? Please describe it here so we can understand the test without needing to refer to a dependency.

Reply:
Sorry, I can't get the point of your comment

Reply:
Ah, I think this comment is using an old name and method (`AMQPWrapper.#connect()`), but my suggestion here is to describe why we have to call it twice. I'll add another suggestion

Reply:
Yes, that's true! I updated the comment with the new class.method name.
The reason why I called it twice is to increase test coverage.
If updating the class.name in the comment is enough, I can resolve this suggestion, otherwise please give me more details.

Reply:
Next suggestion, clarify this. Thank you